content/php/concepts/string-functions/terms/html-entity-decode/html-entity-decode.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: '.html_entity_decode()'
+Description: 'Converts HTML entities to characters.'
+Subjects: 
+- 'Web Design'
+- 'Web Development'
+Tags:
+- 'Functions'
+- 'Characters'
+CatalogContent:
+- 'learn-php'
+- 'paths/computer-science'
+---
+
+**'html_entity_decode()'** is the opposite of **'htmlentities()'** in that it converts HTML entities in the string to their corresponding characters.
+
+## Syntax
+
+```pseudo
+html_entity_decode(str, flags, character-set)
+```
+- **string:** - Specifies the string to decode
+- **flags:** - Specifies how to handle quotes and which document type to use.
+- **character-set:** These are optional strings that specify which character-set to use.
+
+## Example
+
+The below example takes value for `$orig`. Assign to `$a` value of `$orig` using `htmlentities()`. Decodes it using `html_entity_decode()` and assigning it to `$b`. Show values for Both, `$a` and `$b`:
+
+```php
+<?php
+$orig = "dog";
+$a = htmlentities($orig);
+$b = html_entity_decode($a);
+echo $a;
+echo "\n";
+echo $b;
+?>
+```
+
+The above example generates the following output:
+
+```shell
+dog
+dog
+```
+
+## Codebyte Example
+
```

Comment:
Instead of comment within the code, give an introductory sentence summarizing what the code block does.

Reply:
@SSwiniarski Resolved.